rfcs/template.md

```diff
@@ -0,0 +1,56 @@
+# Descriptive Name for the Proposal
+
+## Introduction
+
+Replace the text in this section with a short description of the proposed idea.
+
+Explain the motivation for a proposal, such as:
+- Improved users experience for API changes and extensions. Code snippets to
```

Comment:
Please add an apostrophe to "user's", it's possessive in this case, or change to "user"